mcpscanner/core/static_analysis/native_analyzer.py

```
@@ -3896,6 +3936,356 @@ def _ts_extract_parameters(self, node: "Node") -> List[Dict[str, Any]]:
+        detect their security operations (alias-aware), and OR the resulting
+        flags into the handler, record ``reachable_functions``, and mark any
+        tainted parameter that reaches a dangerous callee as ``reaches_external``.
+        """
+        func_types = self.FUNCTION_NODE_TYPES.get(self.language, set())
+        if not func_types:
+            return
+        try:
+            root = self._ts_root(handler_node)
+            index = self._ts_build_function_index(root, func_types)
+            ambiguous_bare = self._ts_ambiguous_bare_functions(root)
+        except Exception:  # pragma: no cover - defensive
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)